vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -289,6 +289,11 @@ func removeSubnetValidatorValidation(
 		return nil, false, err
 	}
 
+	fee, err := chainState.GetBaseFee()
```

Comment:
Licence header (beginning of file)

Reply:
Thanks, there were one more file without header. Added to both